modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/categorization/tags/MergeTagsModal.tsx

```diff
@@ -27,33 +27,53 @@ export default function MergeTagsModalContent({
 	loadData,
 	tagId,
 	tagName,
-	tagsList,
 }: {
 	closeModal: () => void;
 	loadData: () => {};
 	tagId: number;
 	tagName: string;
-	tagsList: any;
 }) {
 	const [tags, setTags] = useState<Tag[]>([]);
 	const [selectedTags, setSelectedTags] = useState<Tag[]>([]);
 
-	const allTags = useMemo(() => {
-		return tagsList.map((item: any) => ({
-			label: item.name,
-			value: item.tagId,
-		}));
-	}, [tagsList]);
-
 	useEffect(() => {
-		setTags(allTags);
+		const getTags = async () => {
+			const response = await fetch(
```

Comment:
Use the new `ApiHelper` to do the fetch. See https://github.com/liferay-site-management/liferay-portal/pull/2142